microsoft-dracut-config.spec

```diff
@@ -37,26 +37,28 @@ BuildArch:      noarch
 %build
 
 %install
-install -D -m 644 etc/dracut.conf.d/07-nvme.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-nvme.conf
-install -D -m 644 etc/dracut.conf.d/07-ext4.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-ext4.conf
-install -D -m 644 etc/dracut.conf.d/07-virtio.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-virtio.conf
-install -D -m 644 etc/dracut.conf.d/07-xfs.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-xfs.conf
-install -D -m 644 etc/dracut.conf.d/11-resume.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/11-resume.conf
+install -D -m 644 usr/lib/dracut/dracut.conf.d/07-nvme.conf \
+    %{buildroot}/usr/lib/dracut/dracut.conf.d/07-nvme.conf
+install -D -m 644 usr/lib/dracut/dracut.conf.d/07-ext4.conf \
+    %{buildroot}/usr/lib/dracut/dracut.conf.d/07-ext4.conf
+install -D -m 644 usr/lib/dracut/dracut.conf.d/07-virtio.conf \
+    %{buildroot}/usr/lib/dracut/dracut.conf.d/07-virtio.conf
+install -D -m 644 usr/lib/dracut/dracut.conf.d/07-xfs.conf \
+    %{buildroot}/usr/lib/dracut/dracut.conf.d/07-xfs.conf
+install -D -m 644 usr/lib/dracut/dracut.conf.d/11-resume.conf \
+    %{buildroot}/usr/lib/dracut/dracut.conf.d/11-resume.conf
 
 %check
 
 %files
 %doc README.md
 %license LICENSE
-%config %{_sysconfdir}/dracut.conf.d/07-nvme.conf
-%config %{_sysconfdir}/dracut.conf.d/07-ext4.conf
-%config %{_sysconfdir}/dracut.conf.d/07-virtio.conf
-%config %{_sysconfdir}/dracut.conf.d/07-xfs.conf
-%config %{_sysconfdir}/dracut.conf.d/11-resume.conf
+%dir /usr/lib/dracut/
+%dir /usr/lib/dracut/dracut.conf.d/
+%{_libexecdir}dracut/dracut.conf.d/07-nvme.conf
+%{_libexecdir}dracut/dracut.conf.d/07-ext4.conf
+%{_libexecdir}dracut/dracut.conf.d/07-virtio.conf
+%{_libexecdir}dracut/dracut.conf.d/07-xfs.conf
+%{_libexecdir}dracut/dracut.conf.d/11-resume.conf
```

Comment:
`%_libexecdir` expands to `/usr/libexec` in openSUSE and probably SLFO too. It's probably better to `/usr/lib` directly.

Reply:
Fixed. Using full path instead